KeyGenerator/Sources/Encryption.swift

```diff
@@ -4,72 +4,52 @@
 //
 //  Created by Masami on 2023/09/07.
 //
-//  Source
-//  https://github.com/KyleBanks/XOREncryption/blob/master/Swift/XOREncryption.swift
 
+import CryptoKit
 import Foundation
 
 final class Encryption {
 
     static let shared: Encryption = Encryption()
 
-    var encryptionKey: String = "%@"
+    private(set) var encryptionKeyString: String = "%@"
 
-    private var encryptionKeyBytes: [UInt8] {
-        return [UInt8](encryptionKey.utf8)
+    private var encryptionKey: SymmetricKey {
+        SymmetricKey.make(base64EncodedString: encryptionKeyString)
     }
 
     private init() {}
 
+    func makeSymmetricKey() {
+        encryptionKeyString = SymmetricKey(size: .bits256).toString()
+    }
+
     func encrypt(_ input: String) -> String {
-        let inputValueBytes = [UInt8](input.utf8)
-        let encryptionKeyBytes = self.encryptionKeyBytes
-        if inputValueBytes.count > encryptionKeyBytes.count {
-            fatalError("The length of the input value is longer than the encryption key.")
-        }
-        let encryptedBytes: [UInt8] = inputValueBytes.enumerated().map { byte in
-            byte.element ^ encryptionKeyBytes[byte.offset]
-        }
-        guard let encryptedValue = String(bytes: encryptedBytes, encoding: .utf8),
-              let percentEncodedValue = encryptedValue.addingPercentEncoding(withAllowedCharacters: .urlPathAllowed) else {
-            fatalError("Encryption failed")
-        }
-        return percentEncodedValue.base64Encode()
+        let data = input.data(using: .utf8)!
```

Comment:
Force unwrapping should be avoided as possible as to avoid unexpected run-time errors.

> https://realm.github.io/SwiftLint/force_unwrapping.html

And more, should use `Data(input.utf8)` instead of `input.data(using: .utf8)!` because cast the `String` value to `Data` with encoding `UTF-8` is always succeeded in Swift 5 above.

> https://github.com/apple/swift/blob/a353176e1eb570a56809cf4202f5f30aa8905840/stdlib/public/Darwin/Foundation/NSStringAPI.swift#L791-L811

Reply:
@tkgstrator 

> Force unwrapping should be avoided as possible as to avoid unexpected run-time errors.

thank you. I just created this pull request to check the differences. This code is still for verification purposes, so I plan to modify it later.